### PR TITLE
Live events automated ad experiment

### PR DESCRIPTION
**Pull request by buck06191**
This introduces a new ad type that automates the process of creating an
advert for use with masterclasses and live events.

Information is pulled in from a membership API endpoint and this is used
to auto-generate the advert content based on the URL.

At present this points to the live events end point only. See: https://github.com/guardian/membership-frontend/pull/1916

Co-authored-by: Fares Marek Basmadji <fares.basmadji@guardian.co.uk>

### CODE REVIEW

File: src/events-multiple/web/index.html [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+        'weekly': '{{#svg}}guardian-weekly-logo{{/svg}}',
+        'members': '{{#svg}}guardian-members-logo{{/svg}}',
+        'patron': '{{#svg}}guardian-patron-logo{{/svg}}',
+        'lifestyle': '{{#svg}}guardian-generic-logo{{/svg}}',
+        'climate': '{{#svg}}guardian-generic-logo{{/svg}}',
+        'climate2': '{{#svg}}guardian-generic-logo{{/svg}}',
+    };
+</script>
+
+<!-- <a class="blink advert advert--manual advert--prominent-[%IsProminent%] advert--[%Tone%]" href="%%CLICK_URL_UNESC%%[%Offer1URL%]" data-link-name="Offer 1 | [%Offer1Title%]" target="_top">
```
Comment (ioanna0) [inline src/events-multiple/web/index.html]: could we clean up this commented lines? 🧹

File: src/events-multiple/web/index.js [hunk 2]
```diff
@@ -0,0 +1,31 @@
+// import { getIframeId, getWebfonts, reportClicks, resizeIframeHeight, onViewport } from '../../_shared/js/messages';
```
Comment (ioanna0) [inline src/events-multiple/web/index.js]: could we clean up this file's commented lines? 🧹

File: src/_shared/js/events-multiple.js [hunk 3]
```diff
@@ -0,0 +1,160 @@
...
+    "[%Offer4Image%]"
+  ]
+};
+
+const getEventsData = () => {
+  return fetch(`${ENDPOINT}`)
+    .then(response => response.json())
+    .then(json => {
+      const filteredJson = json.events.filter(f => OVERRIDES.urls.includes(f.url));
+      console.log(filteredJson);
```
Comment (ioanna0) [inline src/_shared/js/events-multiple.js]: i assume this console log was left accidentally after testing

File: src/events-multiple/web/index.js (hunk 2 above)
Comment (buck06191) [reply src/events-multiple/web/index.js]: Indeed. I've gone through and got rid of a lot of comments. Thanks for catching it.

**Pull request merged**